bundles/CoreBundle/EventListener/SearchBackendListener.php

```diff
@@ -49,8 +49,8 @@ public static function getSubscribedEvents()
             DocumentEvents::PRE_DELETE => 'onPreDeleteElement',
             AssetEvents::PRE_DELETE => 'onPreDeleteElement',
 
-            DataObjectEvents::POST_UPDATE => 'onPostUpdateElement',
-            DocumentEvents::POST_UPDATE => 'onPostUpdateElement',
+            DataObjectEvents::POST_UPDATE => 'onPostUpdateDataObjectDocument',
+            DocumentEvents::POST_UPDATE => 'onPostUpdateDataObjectDocument',
```

Comment:
A bit strange, that the listener method for `DocumentEvents::POST_UPDATE` is `onPostUpdateDataObjectDocument`.

Reply:
suggestion?